tests/test.nf

```diff
@@ -21,10 +19,27 @@ process RNG {
 
 }
 
+// shows nf-prov behavior with exec tasks
+process EXEC_FOO {
+    tag "${prefix}"
+    publishDir "${params.outdir}", mode: 'copy'
+
+    input:
+    tuple val(prefix), val(constant)
+
+    output:
+    path(outputfile)
+
+    exec:
+    outputfile = new File("${task.workDir}/${prefix}.exec_foo.txt")
```

Comment:
```suggestion
    outputfile = file("${task.workDir}/${prefix}.exec_foo.txt")
```

Reply:
I think the reason this output file wasn't being picked up by nf-prov is because it's a `File` instead of a `Path`. Let me know if this changes fixes it

Reply:
@stevekm are we ok to apply this change?